CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # TRILLIAN Changelog
 
+## HEAD
+
+Not yet released; provisionally v2.0.0 (may change).
```

Comment:
Since this mentions an API change, I guess there should be a major version bump. It does say an "internal" API though, so perhaps third-party storage implementations wouldn't be affected, and so it can be just a minor version bump?

Reply:
I think it would need changes to third party storage but fairly minor ones. I think "internal" here means it doesn't affect end-users.

Reply:
Ok. I think the idea of semantic versioning is that *any* backwards-incompatible change should result in a major version bump, so I think jumping to v2.0.0 is appropriate.

Reply:
Not sure about that as it would only affect code outside this repository.